### PR TITLE
Quickfix for !is being highlighted as keyword when part of a negated variable name.

### LINKED ISSUE #44: !is highlighted as keyword when part of a negated variable name
```kotlin
a = b !is C // type check keyword, all good
a = b ! is C // type check keyword, all good
a.b = !isC // variable name, no-no
a.b = ! isC // split operator fixes it, but I haven't seen this style anywhere
```
Notice how in `!isC` `!is` is highlighted as a keyword.

### PR DESCRIPTION
**Pull request by Animeshz**
Fixes #44

### CODE REVIEW

File: src/keywords.YAML-tmLanguage [hunk 1]
```diff
@@ -2,7 +2,7 @@ repository:
     keywords:
         patterns:
             -
-                match: '(\!in|\!is|as\?)'
+                match: '(\!in|\!is|as\?)\b'
```
Comment (nishtahir) [inline src/keywords.YAML-tmLanguage:5]: I think we probably want the entire token in the word boundary.
```suggestion
                match: '\b(\!in|\!is|as\?)\b'
```

File: test/Keywords.test.kt [hunk 2]
```diff
@@ -69,4 +69,18 @@
 //       ^ source.kotlin
 //        ^ source.kotlin meta.block.kotlin punctuation.section.group.begin.kotlin
   }
-//^ source.kotlin meta.block.kotlin punctuation.section.group.end.kotlin
\ No newline at end of file
+//^ source.kotlin meta.block.kotlin punctuation.section.group.end.kotlin
+
+  val test = !isC
```
Comment (nishtahir) [inline test/Keywords.test.kt:74]: I would also add a regression test for this issue for additional coverage. see the `regression` folder for details on how to do that.

File: src/keywords.YAML-tmLanguage (hunk 1 above)
Comment (Animeshz) [reply src/keywords.YAML-tmLanguage:5]: Probably not, check the third test added.

Comment (nishtahir) [reply src/keywords.YAML-tmLanguage:5]: Ah I see 🤔

**Review by nishtahir — APPROVED** [review]
LGTM 👍

File: test/regression/44.test.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  val test = !isC
+//^^^ source.kotlin storage.type.kotlin
+//   ^^^^^^ source.kotlin
+//         ^ source.kotlin keyword.operator.assignment.kotlin
+//           ^ keyword.operator.logical.kotlin
+//            ^^^ source.kotlin - keyword.operator.kotlin
+
+  val test2 =!isC
+//          ^ keyword.operator.assignment.kotlin
+//           ^^^ - keyword.operator.kotlin
```
Comment (TWiStErRob) [inline test/regression/44.test.kt:12]: @nishtahir I don't get it, this test is testing for the bad case, not what the expectation is. `isC` is a variable name.

Comment (nishtahir) [reply test/regression/44.test.kt:12]: This is testing that the `keyword.operator.kotlin` scope is not applied that's what the `-` sign indicates

Comment (TWiStErRob) [reply test/regression/44.test.kt:12]: Aaaaah 😅

**Pull request merged**